Pcap++/src/PcapLiveDeviceList.cpp

```diff
@@ -26,14 +26,25 @@
 
 namespace pcpp
 {
+	namespace
```

Comment:
why is this anonymous namespace needed?

Reply:
For organizational purposes and to mark the content functions as internal linkage, since they are only used as helpers in the current translation unit.

Reply:
Initially the `fetch*` functions were also as free functions in this namespace, but got changed to static functions as the private ctors of the device classes did not allow construction inside free functions.

Reply:
Then, in this case, I think the convention is to let it be under `pcpp::internal`?

Reply:
Then it would not link internally tho. Anonymous namespaces are special as they specify their linkage as internal. The `internal` namespace is more for things that are used throughout the library (therefor need external linkage) and we don't want the user to use. An anonymous namespace is more like "I don't even want the other parts (translation units) of the library know this exists because they don't need it".

Reply:
I got your intention. Thanks for the explanation.